src/modules/emote_menu/styles/menu.module.css

```diff
@@ -0,0 +1,19 @@
+.divider {
+  margin: 0px;
+}
+
+.header {
+  display: flex;
+  padding: 8px;
+  column-gap: 4px;
+}
+
+.content {
+  display: flex;
+  height: 308px;
```

Comment:
this is a very specific number. how is this obtained? is there a way to get a rounder number?